serial-vault-services-snap/install.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/sh
+
+adduser --disabled-login --disabled-password --gecos "" serialvault
+
+# Install postgresql
```

Comment:
sudo ./install.sh reports the error

pg_ctl: directory "/home/serialvault/snap/postgresql96/common/data" is not a database cluster directory

and after that the services are not started and installation is not ok at all.
I've been trying to execute it with another user (postgres) with same result, even trying to doing it by hand.
In my environment there is no previous postgresql installation.